spec/helpers/your_details_form_helper_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.describe Wpcc::YourDetailsFormHelper, type: :helper do
+  describe '.disable_minimum_contribution_option?' do
+    subject { helper.disable_minimum_contribution_option? }
+
+    context 'salary lower than threshold' do
+      let(:form) do
+        Wpcc::YourDetailsForm.new(age: age,
+                                  gender: gender,
+                                  salary: salary,
+                                  salary_frequency: salary_frequency,
+                                  contribution_preference: pref)
+      end
+
+      before do
+        form.valid?
+        assign(:your_details_form, form)
+      end
+
+      context 'make minimum contribution preference' do
+        let(:age)    { 35 }
+        let(:gender) { 'female' }
+        let(:salary) { 4000 }
+        let(:pref)   { 'minimum' }
+        let(:salary_frequency) { 'year' }
```

Comment:
:neckbeard: this is me being pedantic. the above `{` are all in line. this one isn't. Would prefer one standard, whether thats a single space for all or whether all are inlined